client.go

```diff
@@ -208,6 +224,12 @@ func (c *Client) Connect() error {
 	}
 }
 
+// Userlist returns the userlist for a given channel
+func (c *Client) Userlist(channel string) []string {
```

Comment:
This method should also return an error IMO that signals when a userlist couldn't be found for a channel, since we haven't joined that channel. An empty list would be confusing.